pkg/dcgmexporter/kubernetes.go

```diff
@@ -35,6 +36,8 @@ var (
 	socketPath = socketDir + "/kubelet.sock"
 
 	connectionTimeout = 10 * time.Second
+
+	gkeMigDeviceIdRegex = regexp.MustCompile(`^nvidia(?P<gpu>[0-9]+)/gi(?P<gi>[0-9]+)$`)
```

Comment:
You do not use the group names in the code below, but those make the regex visually more difficult to read. Could we remove the group names or use them in the code below?

Reply:
Yeah, makes sense - removed.